modules/ossm-remove-cleanup.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/installing-ossm.adoc
+
+
+[id="ossm-remove-cleanup_{context}"]
+= Clean up Operator resources
+
+Follow this procedure to manually remove resources left behind after removing the {ProductName} Operator using the OpenShift console.
+
+.Prerequisites
+
+* An account with cluster administration access.
```

Comment:
We gotta remember to add this to https://github.com/openshift/openshift-docs/pull/30066 after this is merged. Nothing to change, just a note.